src/components/Video/Video.js

```diff
@@ -1,31 +1,56 @@
-import Hls from 'hls.js';
-import React, { useCallback, useEffect, useRef, useState } from 'react';
+import React, { useCallback, useEffect } from 'react';
 import { useBeforeUnload } from 'react-router-dom';
+import Hls from 'hls.js';
 
 import { useKeyDown } from '../../../hooks/useKeyDown';
-import { KEY_PLAYBACK_RATE } from '../../../utils/constants';
-
-import VolumeLogo from './img/volume.svg';
+import {
+  KEY_PLAYBACK_RATE,
+  MAX_VIDEO_SPEED,
+  MIN_VIDEO_SPEED,
+  VIDEO_SPEED_STEP,
+} from '../../../utils/constants';
```

Comment:
I recommend you to avoid such long "dot" path imports. 
You can try to configure your project to use '@' to get paths from the root folder.

I see that your project uses JavaScript, but here is the solution for your case in TypeScript:
https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json